build-system/compile/bundles.config.extensions.json

```diff
@@ -363,7 +363,8 @@
     "version": "1.0",
     "latestVersion": "0.1",
     "options": {
-      "hasCss": true
+      "hasCss": true,
+      "npm": true
```

Comment:
This `npm: true` should be moved to `amp-facebook`. The `amp-facebook-*` components now rely on a single `Facebook` Preact component which lives inside the `amp-facebook` directory.

Reply:
Moved in 0a0f28e561e292b115792728a3e03196226839a6.